frontend/src/components/pages/CampOverview/MoveCampersModal.tsx

```diff
@@ -0,0 +1,256 @@
+            );
+            setModalPage(1);
+          } else {
+            setCampers([camper]);
+            setModalPage(2);
+          }
+        },
+      );
+
+      CampsAPIClient.getCampById(campSession.camp).then((camp) => {
+        setCampSessions(camp.campSessions);
+      });
+    }
+  }, [camper, moveCamperModalIsOpen, campSession.camp]);
+
+  const displayModalContent = () => {
+    if (modalPage === 1) {
+      return (
+        <>
+          <ModalHeader>Move group members?</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <p>
+              {camper.firstName} {camper.lastName} was registered with the
+              following group members. Please select all group members that you
+              would like to move as well:
+            </p>
+            <Stack
+              direction="column"
+              alignItems="flex-start"
+              mt="20px"
+              maxH="60vh"
+              overflowY="scroll"
+            >
+              <Checkbox
+                className="camper-checkbox"
+                isChecked={camperIsSelected.every(Boolean)}
+                onChange={() => {
+                  if (!camperIsSelected.every(Boolean)) {
+                    setCamperIsSelected(
+                      Array<boolean>(campers.length).fill(true),
+                    );
+                  } else {
+                    setCamperIsSelected(
+                      Array<boolean>(campers.length).fill(false),
+                    );
+                  }
+                }}
+              >
+                Select all
+              </Checkbox>
+              {campers.map((campersItem, i) => (
+                <Checkbox
+                  className="camper-checkbox"
+                  key={i}
+                  isChecked={camperIsSelected[i]}
+                  onChange={() => {
+                    setCamperIsSelected(
+                      camperIsSelected.map((camperItem, camperIdx) => {
+                        if (camperIdx === i) {
+                          return !camperItem;
+                        }
+                        return camperItem;
+                      }),
+                    );
+                  }}
+                >{`${campersItem.firstName} ${campersItem.lastName}`}</Checkbox>
+              ))}
+            </Stack>
+          </ModalBody>
+
+          <ModalFooter>
+            <Button onClick={moveCamperModalOnClose}>Cancel</Button>
+            <Button
+              variant="solid"
+              style={{
+                backgroundColor: "#468740",
+                color: "white",
+                marginLeft: "10px",
+              }}
+              onClick={() => {
+                setModalPage(2);
+              }}
+              disabled={
+                camperIsSelected.reduce<number>(
+                  (prev, cur) => (cur ? prev + 1 : prev),
+                  0,
+                ) === 0
```

Comment:
An admin can choose not to move any grouped campers so this condition is not accurate. I don't think the next button can ever be disabled for this page